### PR TITLE
Card styling

### PR DESCRIPTION
**Pull request by pvinthemix**
Most of the styling was done on the last push which I forgot to switch branches as mentioned earlier.

### CODE REVIEW

**Comment by pvinthemix** [comment]
We may want to think about having a main.scss file that imports are component css files.

File: src/styles/Card.scss [hunk 1]
```diff
@@ -8,34 +11,30 @@
...
-
 .distance-roundtrip-text {
-  color: #FFFF00;
+  color: $card-text-color;
 }
 
-
 .difficulty-rating-text {
-  color: #FFFF00;
+  color: $card-text-color;
```
Comment (mike-duke) [inline src/styles/Card.scss:39]: good idea for implementing the sass variable!

File: src/styles/Card.scss [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
 }
 
-
 .distance-difficulty-container {
   display: flex;
   justify-content: space-between;
+  font-size: 20px;
 }
 
 .trail-name-title {
```
Comment (thatPamIAm) [inline src/styles/Card.scss:24]: Duplicated code between the classes of `trail-name-title`, park-name-title`, `distance-roundtrip-text`... could DRY this up

Comment (pvinthemix) [reply src/styles/Card.scss:24]: Got it. Thanks @thatPamIAm

**Pull request merged**